### DISCUSSION TITLE
How can I do the installation with Bazel

### DISCUSSION THREAD

**Discussion by meftunca** [uNetworking/uWebSockets#1740] [General]
I am developing my project with Bazel. It is really easy to use. Especially for someone like me who develops on languages like Golang, Typescript and nodejs. I will be very happy if you share an example about it or help me move forward.

In addition, including it as a bazel module in the official bazel registry will make your project spread faster.

**Comment by uNetworkingAB** [comment]
This verbatim request has been made for CMake, Conan, Meson, node-gyp, Ninja before.

The only possible solution is to reject them all and only provide the standards compliant source code, for everyone to consume as they see fit. There are many other reasons but this is the TLDR

**Reply by meftunca** [reply]
Thank you for the return. However, it provides a simple github application to your bazel repo to automatically publish updates. For others you might need a configuration that requires long-term maintenance, but for bazel it's really simple.
Github App: https://github.com/apps/publish-to-bcr
If you still refuse, please allow me to fork your repo and activate this service. If you still refuse, please allow me to fork your repo and activate this service. In this case, my replicated repo will appear as the project repo in registry.bazel

**Reply by uNetworkingAB** [reply]
It's not a refusal it's just that I have 0 interest in any one build system / central repository. This repo is standards compliant C++17, it can be used in any way people need to. I know companies that use it with CMake, Bazel, Meson and whatnot.

> A GitHub app that mirrors releases of your Bazel ruleset to the **Bazel Central Registry**.

It reminds me of Gosbank. One bank to rule them all.
